packages/aws-cdk/test/context.test.ts

```diff
@@ -137,3 +137,18 @@ test('transient values arent saved to disk', async () => {
   // THEN
   expect(config2.context.get('some_key')).toEqual(undefined);
 });
+
+test('transient keys arent saved to disk', async () => {
```

Comment:
Not sure about the purpose of this test since it doesn't use `disableContextCaching`?

Reply:
This is a copy of the test above it ("transient values arent saved to disk") but it checks that the specified keys/values don't get saved to the context, regardless of what the value is.